src/components/WebexMeeting/WebexMeeting.jsx

```diff
@@ -113,13 +115,15 @@ export default function WebexMeeting({
 
 WebexMeeting.propTypes = {
   className: PropTypes.string,
+  controls: PropTypes.func,
   logo: PropTypes.node,
   meetingID: PropTypes.string,
   style: PropTypes.shape(),
 };
 
 WebexMeeting.defaultProps = {
   className: '',
+  controls: undefined,
```

Comment:
Let's have the default for `WebexMeeting` be [what is on `WebexMeetingControlBar` right now](https://github.com/webex/components/blob/master/src/components/WebexMeetingControlBar/WebexMeetingControlBar.jsx#L218).
Simplify `WebexMeetingControlBar` defaults to:
```
(isActive) => (
    isActive
      ? ['mute-audio', 'mute-video', 'leave-meeting']
      : ['mute-audio', 'mute-video', 'join-meeting']